frame/democracy/src/tests.rs

```diff
@@ -0,0 +1,374 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The crate's tests.
+
+use super::*;
+use crate as pallet_democracy;
+use codec::Encode;
+use frame_support::{
+	assert_noop, assert_ok, ord_parameter_types, parameter_types,
+	traits::{Contains, Everything, GenesisBuild, OnInitialize, SortedMembers},
+	weights::Weight,
+};
+use frame_system::{EnsureRoot, EnsureSignedBy};
+use orml_traits::parameter_type_with_key;
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BadOrigin, BlakeTwo256, IdentityLookup},
+	Perbill,
+};
+
+mod cancellation;
+mod decoders;
+mod delegation;
+mod external_proposing;
+mod fast_tracking;
+mod lock_voting;
+// mod preimage;
+// mod public_proposals;
+// mod scheduling;
+// mod voting;
```

Comment:
Why are all these tests disabled?

Reply:
Nothing in specific, these were commented out during troubleshooting, and got pushed into repo. Will have it removed, and make sure these tests passthrough.

Reply:
This is fixed.